prow/scripts/cluster-integration/kyma-integration-gardener.sh

```diff
@@ -122,6 +122,12 @@ gardener::provision_cluster
 # uses previously set KYMA_SOURCE
 if [[ "${KYMA_ALPHA}" == "true" ]]; then
   kyma::alpha_deploy_kyma
+  if [[ "${KYMA_ALPHA_DELETE}" == "true" ]]; then
+    sleep 30
```

Comment:
why do we need this and 2 lines below sleep?

Reply:
@strekm We want to: install Kyma, then uinstall it, then install iKyma again and run tests.

Reply:
And because the script `prow/scripts/cluster-integration/kyma-integration-gardener.sh` is used by e.g. pre/post submit jobs where we don't want to test entire scenario only simpler install/test